x-pack/plugins/security_solution/public/common/components/line_clamp/index.tsx

```diff
@@ -64,6 +58,15 @@ const LineClampComponent: React.FC<{ content?: string | null }> = ({ content })
     return null;
   }
 
+  const StyledLineClamp = styled.div`
```

Comment:
Hey! Passing styled components inside a component should be [avoid](https://styled-components.com/docs/faqs#why-should-i-avoid-declaring-styled-components-in-the-render-method) it as it creates performance issues. You can still pass props to a styled component. Example:

```
const StyledLineClamp = styled.div<{ lineClampHeight: string }>`
max-height: ${p => (`${p.lineClampHeight)em`}};
`


<StyledLineClamp lineClampHeight={lineClampHeight} />`
```

Reply:
Good point! I didn't know that styled-components could get parameters. Thank you!
I pushed an extra commit to fix it.